screens/HomeScreen/HomeScreen.js

```diff
@@ -24,45 +25,97 @@ function HomeScreen(props) {
   return (
     <>
       <Header title='Inicio' />
-      <View style={styles.screen}>
-        <Avatar
-          size='large'
-          rounded
-          source={{
-            uri:
-              'https://www.nicepng.com/png/detail/804-8049853_med-boukrima-specialist-webmaster-php-e-commerce-web.png',
-          }}
-        />
-        <Badge
-          value={apiKey ? 'Sincronizado con Buda' : 'Falta Sincronizar'}
-          status={apiKey ? 'success' : 'error'}
-        />
-        <Text>{`Hola ${email}!`}</Text>
+      <ThemeProvider theme={Theme}>
+        <View style={styles.screen}>
+          <Avatar
+            containerStyle={styles.avatar}
+            source={require('../../assets/Images/spacemonkey.png')}
+          />
+          <Text style={styles.nameText}>Astronaut Monkey</Text>
+          <Text style={styles.emailText}>{`${email}`}</Text>
+          <Text style={styles.walletText}>Wallet BitSplit</Text>
+          <Button
+            title='Editar'
+            type='outline'
+            buttonStyle={styles.button}
+            titleStyle={styles.buttonText}
+          />
 
-        {balance ? (
-          <View>
-            <Text
-              style={styles.saldoText}
-            >{`Saldo \n${balance.BTC.amount} BTC\n${balance.CLP.amount} CLP `}</Text>
-            <Button
-              title='Generar Pago'
-              type='solid'
-              onPress={() => props.navigation.navigate('Pagar')}
-            />
-          </View>
-        ) : (
-          <View>
-            <Text style={styles.saldoText}>
-              {loading || 'Debes sincronizar con Buda'}
-            </Text>
-            <Button
-              title='Sincronizar'
-              type='solid'
-              onPress={() => props.navigation.navigate('Buda')}
-            />
-          </View>
-        )}
-      </View>
+          {apiKey ? (
+            <View style={styles.wallet}>
+              <Text style={styles.saldoText}>
+                Saldo: ${balance.BTC.amount} BTC
+              </Text>
+            </View>
+          ) : (
+            <View style={styles.wallet}>
+              <Text style={styles.saldoText}>
+                {loading || 'Debes sincronizar con Buda'}
```

Comment:
Esto cuando loading sea true va a mostrar un "true" en el Text. Deberías poner esto:
```suggestion
                {loading ? 'Cargando...' : 'Debes sincronizar con Buda'}
```